src/node-project.ts

```
@@ -171,10 +171,22 @@ export interface NodeProjectOptions extends ProjectOptions, NodePackageOptions,
    * Automatically merge projen upgrade PRs when build passes.
    * Auto-approves the pull request which mergify will then merge.
    *
-   * @default - "true" if mergify auto-merge is enabled (default)
+   * Only applies if `autoApproveOptions` are defined.
+   *
+   * @default - true
    */
   readonly projenUpgradeAutoMerge?: boolean;
 
+  /**
+   * Automatically merge deps upgrade PRs when build passes.
+   * Auto-approves the pull request which mergify will then merge.
+   *
+   * Only applies if `autoApproveOptions` are defined.
```

Should this fail if autoApprove is not configured?

---

Yes. Changed.